DogeNet.IdentityServer/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model DogeNet.IdentityServer.ViewModels.LoginViewModel
+
+<head>
+    <link href=@Url.Content("~/styles/app.css") rel="stylesheet" type="text/css"/>
+    
+    </head>
+<div class="container-login">
+    <div class="wrap-login">
+        <span class="login-form-title p-b-26">Login</span>
+        <form asp-controller="Account" asp-action="Login" method="post">
+            <div asp-validation-summary="All"></div>
+
+            <input hidden name="ReturnUrl" />
+            <div class="wrap-input">
+                <input class="input" placeholder="Email" autoFocus required name="UserName" />
+                <span asp-validation-for="UserName"></span>
+            </div>
+            <div class="wrap-input">
+                <input class="input" placeholder="Password" type="password" required name="Password" />
+                <span asp-validation-for="Password"></span>
+            </div>
+
+            <p>
+                <button class="gradient-button" type="submit">Login</button>
+            </p>
+
```

Comment:
Code styles?

Reply:
@oleh-leobit 
Code style anylizer added to project, all warnings are solved